ads/contentad.js

```diff
@@ -33,10 +33,12 @@ export function contentad(global, data) {
   window.document.body.appendChild(cadDiv);
 
   /* Capture or pass URL */
+  const h = window.context.location.host;
+  const od = (data.url) ? data.url : window.atob(data.d) ;
```

Comment:
can you give a better name to od?

Also, can be simplified as: `const yourNewName = data.url || window.atob(data.d)`